workflow_calcium_imaging/paths.py

```diff
@@ -3,18 +3,30 @@
 
 
 def get_imaging_root_data_dir():
```

Comment:
Can this type suggest pathlib.Path?

```suggestion
def get_imaging_root_data_dir() -> SOMETHING:
```

Reply:
It returns either pathlib.Path or None.